ui/admin/app/components/form/app-token/index.hbs

```diff
@@ -0,0 +1,108 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+<Rose::Form>
+  <Hds::Form::TextInput::Field
+    @value={{@model.name}}
+    @type='text'
+    disabled={{true}}
```

Comment:
This should be readonly. Screen readers will ignore disabled text.
```suggestion
    readOnly={{true}}
```
This is actually something we are working to fix in our codebase for existing inputs but for new we want to use readOnly where we can.